indicators/bancolombia-jr5mnv.yml

```diff
@@ -0,0 +1,41 @@
+title: Bancolombia Phishing Kit jr5mnv
+description: |
+    Detects a phishing kit targeting Bancolombia.
+    This was found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://urlscan.io/result/b4501b15-b767-4c10-a11e-898ae8cf01a7/
+    - https://urlscan.io/result/185193df-05e6-40ff-aa2e-694098a4f03a/
+    - https://urlscan.io/result/2759a513-0dee-49e5-8dfb-820b1acea250/
+    - https://urlscan.io/result/1d52da2c-126c-4a99-bfde-c8a11c2a1a12/
+    - https://urlscan.io/result/e577760c-3901-4fd3-93c4-58a9e2613250/
+    - https://urlscan.io/result/500c35f2-ca7e-4673-8cf0-19a5f686547f/
+
+detection:
+
+    bootstrap:
+      html|contains:
+        - link rel="stylesheet" href="bootstrap.min.css"
+
+    containers:
+      html|contains|all:
+        - div class="limiter"
+        - div class="imgtopd"
```

Comment:
I think both the `bootstrap` and `containers` detection fields can be removed for a more accurate detection which involves checking the CSS contents of the website and seeing if they include the two imgur URLs that are being used in this kit across several different domains

The field could look something like the following:
```yaml
icons:
  css|contains|all:
    - 'https://i.imgur.com/8Ezt3Uy.png'
    - 'https://i.imgur.com/KROsJQP.png'
```